src/Our.Umbraco.Ditto/Ditto.cs

```diff
@@ -135,5 +140,26 @@ public static void RegisterDefaultProcessorType<TProcessorAttributeType>()
         {
             TypeDescriptor.AddAttributes(typeof(TObjectType), new TypeConverterAttribute(typeof(TConverterType)));
         }
+
+        /// <summary>
+        /// Registers a global umbraco application context accessor.
+        /// </summary>
+        /// <typeparam name="TUmbracoApplicationContextAccessorType">The type of the accessor.</typeparam>
+        public static void RegisterUmbracoApplicationContextAccessor<TUmbracoApplicationContextAccessorType>()
```

Comment:
@leekelleher `UmbracoPickerAttribute` is the last reference to those `Convert___FromInt` methods. Definintely happy to drop them as `UmbracoPickerAttribute` does everything